woocommerce/woocommerce-android
# Order creation status selection

Summary
==========
Closes issue #5225 by implementing the Order status view and selection for the Order Creation form, but also defining a clear separation between the Form ViewModel and the shared ViewModel who will contain the Order Draft to be controlled by the entire Order Creation views flow.

Screenshots
==========
| Order Creation status view when the form is first presented | Selecting an Order Status | Order Status after selection changes |
| ------------- | ------------- | ------------- |
| ![Screenshot_20211203_041905](https://user-images.githubusercontent.com/5920403/144562273-5218fe2f-4a85-4bed-b103-46a0ba2798c4.png) | ![Screenshot_20211203_041908](https://user-images.githubusercontent.com/5920403/144562290-a217a735-c5a8-4677-85aa-91109d51fc81.png) | ![Screenshot_20211203_041914](https://user-images.githubusercontent.com/5920403/144562311-18e8c4c0-2750-4c98-8f2c-28acfb39dd8f.png) |

How to Test
==========
1. Make sure the Order Creation experimental flag is activated
2. Go to the Order List and hit the FAB, select the Order Creation option if asked
3. Once inside the Order Creation flow, verify if the initial status is `Pending Payment`
4. Interact with the Edit button and verify if the Status is edited correctly

Update release notes:
